docs/upgrades/automated.md

```diff
@@ -45,7 +45,7 @@ When attempting to upgrade to a new version of K3s, the [Kubernetes version skew
 ### Install the system-upgrade-controller
  The system-upgrade-controller can be installed as a deployment into your cluster. The deployment requires a service-account, clusterRoleBinding, and a configmap. To install these components, run the following command:
 ```bash
-kubectl apply -f https://github.com/rancher/system-upgrade-controller/releases/latest/download/system-upgrade-controller.yaml
+kubectl apply -f https://raw.githubusercontent.com/rancher/system-upgrade-controller/master/manifests/system-upgrade-controller.yaml
```

Comment:
I'm confused, why would it be better to suggest installing from the master branch instead of a release artifact?

Reply:
Sorry, when I was going through the docs, I was getting a 404 from that URL. It seems to be functional now. Disregard my PR.